newIDE/app/src/MainFrame/Providers.js

```diff
@@ -25,6 +26,12 @@ import {
 import GDevelopThemeContext from '../UI/Theme/ThemeContext';
 import { UnsavedChangesContextProvider } from './UnsavedChangesContext';
 import { CommandsContextProvider } from '../CommandPalette/CommandsContext';
+import { create } from 'jss';
+import rtl from 'jss-rtl';
+
+const jss = create({
```

Comment:
Let's add a quick comment: 
`// Add the rtl plugin to the JSS instance to support RTL languages in material-ui components.`